astropy/cosmology/parameter.py

```diff
@@ -46,40 +49,43 @@ class Parameter:
 
     _registry_validators = {}
 
-    def __init__(self, fvalidate="default", doc=None, *,
-                 unit=None, equivalencies=[], fmt=".3g", derived=False):
+    def __init__(self, fvalidate="default", doc=None, *, name=None,
```

Comment:
If you change order in the description above, it should be done here too.

Reply:
Ah. Yes. Caught me vacillating between changing the parameter order. In many ways I would prefer all the arguments to be keyword-only. Much easier to add arguments and maintain logical grouping without impacting argument order.
The current ordering is a legacy from modeling this on ``property``, which is a bit unfortunate since the only shared argument is ``doc``!